src/quick-lint-js/fe/lex.cpp

```diff
@@ -996,7 +996,12 @@ void Lexer::skip_less_less_as_less() {
 void Lexer::skip_as_greater() {
   switch (this->last_token_.type) {
   case Token_Type::greater_equal:
-    this->last_token_.type = Token_Type::equal;
+    if (this->input_[0] == '>') {
+      this->last_token_.type = Token_Type::equal_greater;
```

Comment:
Nit: I think a comment would be helpful:
```suggestion
      // >=>  ➤  > =>
      this->last_token_.type = Token_Type::equal_greater;
```